.github/workflows/build.yml

```diff
@@ -264,6 +264,12 @@ jobs:
           path: ${{ runner.temp }}
           pattern: digests-${{ matrix.arch.name }}
 
+      - name: Create Google Drive Credentials File
+        run: |
+          mkdir ./secrets
+          google_service_account_json_secret_file=./secrets/test_google_service_account.json
```

Comment:
Not sure how to make this DRYier - but there is duplicate config for the different targets.

Reply:
I'm not sure this is necessary. These images don't actually run tests, they just run the scripts:
- ./docker/ci/test_webapp.sh webapp
- ./docker/ci/test_scripts.sh scripts

and i don't think it will make much difference if the keys are configured or not to run these scripts

Reply:
Without this configuration,  the docker compose command fails because the google service config file is missing when the docker tries to bind the secrets.